forloop_common_structures/__init__.py

```diff
@@ -0,0 +1,3 @@
+from forloop_common_structures import *
+
+__all__ = ["core"]
```

Comment:
I would be happier if we still include "core" in the imports - over time its much easier to refactor when something gets moved than if we add everything to __all__